netsim/augment/nodes.py

```diff
@@ -469,9 +481,16 @@ def default_static_route_list(topology: Box) -> list:
 * The IPv4 static routes are generated if the node uses IPv4 AF
 * The IPv6 static routes are generated if the node uses IPv6 AF and does not listen
   to Router Advertisements
+
+Hosts that have management VRF will get the default static routes, other hosts will
+get static routes configured in the 'routing.static.host' parameter or generated from
+the address pools.
 '''
 def add_host_static_routes(topology: Box) -> None:
   sr_list = topology.get('routing.static.host',None)
```

Comment:
default ```[]```?

Reply:
You might want to have no static routes on the hosts, in which case `[]` might come handy as a valid value.